specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1437,6 +1437,132 @@
                     }
                 }
             }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/allowedConnections": {
+            "get": {

```

Comment:
Are these resources system-generated? The user can't update/delete them?

Reply:
Correct. It is a read only collection. User get only see the outcome of the computation we perform.

Reply:
@KrisBash , This API was reviewed by Gaurav more than 6m ago and it is already deployed. We issued the PR here in order to add it to ASC SDK and to update documentation.